src/utils/validation/project-member-schema.ts

```diff
@@ -10,36 +10,30 @@ export const createProjectMemberSchema = object().shape({
       const end_date = this.parent.end_date
       return new Date(start_date) <= new Date(end_date)
     })
-    .test(
-      "start-date",
-      "Must select a date that is within the valid range.",
-      function (start_date) {
-        const currentDate = new Date()
+    .test("start-date", "Start date and", function (start_date) {
+      const currentDate = new Date()
 
-        const maxDate = new Date(
-          currentDate.getFullYear() + 50,
-          currentDate.getMonth(),
-          currentDate.getDate()
-        )
+      const maxDate = new Date(
+        currentDate.getFullYear() + 50,
+        currentDate.getMonth(),
+        currentDate.getDate()
+      )
 
-        const minDate = new Date(
-          currentDate.getFullYear() - 50,
-          currentDate.getMonth(),
-          currentDate.getDate()
-        )
+      const minDate = new Date(
+        currentDate.getFullYear() - 50,
+        currentDate.getMonth(),
+        currentDate.getDate()
+      )
 
-        return (
-          new Date(start_date) <= new Date(maxDate) && new Date(minDate) <= new Date(start_date)
-        )
-      }
-    ),
+      return new Date(start_date) <= new Date(maxDate) && new Date(minDate) <= new Date(start_date)
+    }),
   end_date: string()
     .required("End date is required")
     .test("end-date", "End date must not be earlier than start date.", function (end_date) {
       const start_date = this.parent.start_date
       return new Date(end_date) >= new Date(start_date)
     })
-    .test("end-date", "Must select a date that is within the valid range.", function (end_date) {
```

Comment:
Don't remove this validation, this is for when range of date is more than or less than 50 years ago.

Reply:
It's not removed; I changed the syntax, but the validation for 50 years is still in place.